packages/frontend/src/hooks/useProject.ts

```diff
@@ -25,7 +25,7 @@ const getProject = async (id: string) =>
         body: undefined,
     });
 
-export const useProject = (id: string) => {
+export const useProject = (id: string | undefined) => {
```

Comment:
What happens if I `getProject('')`? 404?

Reply:
It shouldn't happen since we only enable the useQuery if the `id` is defined.